src/react-components/profile-entry-panel.js

```diff
@@ -87,6 +87,7 @@ class ProfileEntryPanel extends Component {
                 onFocus={e => e.target.select()}
                 onChange={e => this.setState({ displayName: e.target.value })}
                 required
+                spellCheck="false"
```

Comment:
looks like someone needs to install the ornithology pack to their system dictionary